src/lib/routing-tables.js

```diff
@@ -0,0 +1,70 @@
+'use strict'
+const routing = require('five-bells-routing')
+
+/**
+ * Maintain two sets of `routing.RoutingTables`.
```

Comment:
Is there a way to modify the code in `five-bells-routing` to make the raw vs shifted option configurable? Seems like that would be better than copying over this code here